### PR TITLE
chore: minor updates to accomodate new structure

### PR DESCRIPTION
**Pull request by AfaqShuaib09**
This PR updates the key values for newly added attributes from `snake_case` to `camel_case`

`future_variants` --> `futureVariants`
`custom_presentations` --> `CustomPresentations`

### CODE REVIEW

File: course_discovery/apps/course_metadata/management/commands/tests/test_populate_executive_education_data_csv.py [hunk 1]
```diff
@@ -148,11 +148,11 @@ class TestPopulateExecutiveEducationDataCsv(CSVLoaderMixin, TestCase):
...
     SUCCESS_API_RESPONSE_CUSTOM_AND_FUTURE_VARIANTS['products'][0].update({
-        'custom_presentations': [{**copy.deepcopy(variant_1), 'websiteVisibility': 'private', 'status': 'active'}],
-        'future_variants': [
+        'customPresentations': [{**copy.deepcopy(variant_1), 'websiteVisibility': 'private', 'status': 'active'}],
+        'futureVariants': [
             {
-                **copy.deepcopy(variant_2), 'websiteVisibility': 'public', 'status': 'scheduled',
-                'startDate': '2026-03-20', 'endDate': '2026-04-28', 'finalRegCloseDate': '2026-03-26'
+                **copy.deepcopy(variant_2), 'websiteVisibility': 'public', 'startDate': '2026-03-20',
+                'endDate': '2026-04-28', 'finalRegCloseDate': '2026-03-26', 'status': 'active'
```
Comment (DawoudSheraz) [inline course_discovery/apps/course_metadata/management/commands/tests/test_populate_executive_education_data_csv.py]: why is the status changed to active from scheduled?

Comment (AfaqShuaib09) [reply course_discovery/apps/course_metadata/management/commands/tests/test_populate_executive_education_data_csv.py]: ah, I did some changes for local testing, lemme revert this

Comment (DawoudSheraz) [reply course_discovery/apps/course_metadata/management/commands/tests/test_populate_executive_education_data_csv.py]: Ideally, the tests should have failed due to this. It means the tests need more strict checks.

Comment (AfaqShuaib09) [reply course_discovery/apps/course_metadata/management/commands/tests/test_populate_executive_education_data_csv.py]: Sure, lemme update the test case to cater this

Comment (AfaqShuaib09) [reply course_discovery/apps/course_metadata/management/commands/tests/test_populate_executive_education_data_csv.py]: I've updated the test case with a strict check, now switching the status attribute will fail the testcase

**Review by DawoudSheraz — APPROVED** [review]

**Review by hamza-56 — APPROVED** [review]

**Pull request merged**